formatter/action.yml

```diff
@@ -0,0 +1,21 @@
+name: "Formatter"
+description: "Format filenames into the format acceptable by TheAlgorithms opganisation"
+author: "tjgurwara99"
```

Comment:
Hmm. Maybe we should set the author to `TheAlgorithms` as multiple people have contributed to this script.